pkg/security/secl/rules/ruleset.go

```diff
@@ -29,9 +28,11 @@ type CombinePolicy = string
 
 // Combine policies
 const (
-	NoPolicy       CombinePolicy = ""
-	MergePolicy    CombinePolicy = "merge"
-	OverridePolicy CombinePolicy = "override"
+	NoPolicy               CombinePolicy = ""
+	MergePolicy            CombinePolicy = "merge"
+	OverridePolicy         CombinePolicy = "override"
+	RuleSetTagKey                        = "ruleset"
```

Comment:
nit: I would create another const section this one seems to be used to define rule operations